backend/src/main/kotlin/app/ehrenamtskarte/backend/verification/webservice/schema/CardMutationService.kt

```diff
@@ -70,7 +70,7 @@ class CardMutationService {
             CardRepository.insert(
                 hashedCardInfo,
                 activationSecretHash,
-                cardInfo.expirationDay.toLong(),
+                expirationDay = if (cardInfo.hasExpirationDay()) cardInfo.expirationDay.toLong() else null,
```

Comment:
To make it consistent with the other optional fields:
```suggestion
                expirationDay = cardInfo.expirationDayOrNull?.toLong(),
```

Also, the same must be done for `createStaticVerificationCode`

Reply:
updated pr

Reply:
@f1sh1918 I don't think so 🤔 at least I cannot see it yet.

Reply:
sorry ssh timeout :/ gonna push it soon

Reply:
@michael-markl now its pushed